src/palakSJun20/ExtractItemIdFromUrl.java

```diff
@@ -0,0 +1,50 @@
+/*Extract ItemId from below input
+
+	String input = https://www.facebook.com/technocredits/temId=12345/
+	String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+	String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+	String input = https://www.facebook.com/technocredits/empid=123/
+
+	Output : 
+	item id is 12345 
+	        OR
+	no item id present.*/
+package palakSJun20;
+
+public class ExtractItemIdFromUrl {
+	void findIdFromUrl(String input) {
+		String output = "";
+		if (input.contains("itemId")) {
+			for (int index = input.indexOf("itemId"); index < input.length(); index++) {
+				if (input.charAt(index) >= '0' && input.charAt(index) <= '9') {
+					output += input.charAt(index);
+					if (!(input.charAt(index + 1) >= '0' && input.charAt(index) <= '9')) {
```

Comment:
Here if is not required, you can write else block for parent if.

if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}